packages/shared/src/utils/date/date-time.ts

```diff
@@ -274,3 +275,24 @@ export const convertTimeFormat = (time: string) => {
     const time_suffix = `${Number(time_hour) >= 12 ? 'pm' : 'am'}`;
     return `${formatted_time} ${time_suffix}`;
 };
+
+/**
+ *  Get a formatted time since a timestamp.
+ * @param  {Number} timestamp in ms
+ * @return {String} '10s ago' or '1m ago' or '1h ago' or '1d ago' or '1 Jan 2019'
+ */
+export const getTimeSince = (timestamp: number) => {
+    if (!timestamp) return '';
+    const seconds_passed = Math.floor((Date.now() - timestamp) / 1000);
+
+    if (seconds_passed < 60) {
+        return localize('{{seconds_passed}}s ago', { seconds_passed });
+    }
+    if (seconds_passed < 3600) {
+        return localize('{{minutes_passed}}m ago', { minutes_passed: Math.floor(seconds_passed / 60) });
+    }
+    if (seconds_passed <= 86400) {
```

Comment:
```suggestion
    if (seconds_passed < 86400) {
```
Should this be just `<` check in order to be consistent with the other checks?

Reply:
@balakrishna-deriv yeah, i took the logic from bot, but I can see that 86400 seconds is actually 24 hours, sure, will fix it👍🏻 thank you)